src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length - 1);
```

Comment:
make a global Random field and use it here, so you won't need to create every time an instance of Random